.github/workflows/build.yml

```diff
@@ -100,6 +100,14 @@ jobs:
           labels: |
             org.opencontainers.image.documentation=https://docs.nginx.com/nginx-gateway-fabric
             org.opencontainers.image.vendor=NGINX Inc <kubernetes@nginx.com>
+          annotations: |
+            org.opencontainers.image.documentation=https://docs.nginx.com/nginx-gateway-fabric
+            org.opencontainers.image.vendor=NGINX Inc <kubernetes@nginx.com>
```

Comment:
Does this string value work properly with whitespace since it's not wrapped in quotes?

Reply:
yeah it takes everything that is on a line. I've added the output of the manifest now after we build the image so we can see what's in there.